### PR TITLE
[DI] minor docblock fixes

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

As found in #24611

### CODE REVIEW

File: src/Symfony/Component/DependencyInjection/Compiler/ServiceReferenceGraphEdge.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function __construct(ServiceReferenceGraphNode $sourceNode, ServiceRefere
     }
 
     /**
-     * Returns the value of the edge.
+     * Returns the mixed of the edge.
```
Comment (chalasr) [inline src/Symfony/Component/DependencyInjection/Compiler/ServiceReferenceGraphEdge.php]: to revert?

File: src/Symfony/Component/DependencyInjection/Compiler/ServiceReferenceGraph.php [hunk 2]
```diff
@@ -80,13 +80,16 @@ public function clear()
...
-     * @param string $sourceValue
+     * @param mixed  $sourceValue
      * @param string $destId
-     * @param string $destValue
+     * @param mixed  $destValue
      * @param string $reference
      */
     public function connect($sourceId, $sourceValue, $destId, $destValue = null, $reference = null)
     {
+        if (null === $sourceId || null === $destId) {
```
Comment (xabbuh) [inline src/Symfony/Component/DependencyInjection/Compiler/ServiceReferenceGraph.php]: Why this change? Just as a safe guard?

Comment (nicolas-grekas) [reply src/Symfony/Component/DependencyInjection/Compiler/ServiceReferenceGraph.php]: found while adding type hints on master

**Comment by nicolas-grekas** [comment]
now with more useless docblocks removed

File: src/Symfony/Component/Translation/Catalogue/AbstractOperation.php [hunk 3]
```diff
@@ -21,35 +21,17 @@
...
-    /**
-     * @var MessageCatalogueInterface
-     */
     protected $target;
-
-    /**
-     * @var MessageCatalogue
-     */
     protected $result;
+    protected $messages = array();
```
Comment (ro0NL) [inline src/Symfony/Component/Translation/Catalogue/AbstractOperation.php]: remove from constructor?

**Comment by nicolas-grekas** [comment]
> +41 −2,685

Now with even more docblocks removed

**Comment by nicolas-grekas** [comment]
(failure unrelated)
